homeassistant/components/weatherkit/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "WeatherKit Setup",
+        "description": "Enter your location details and WeatherKit authentication credentials below.",
+        "data": {
+          "name": "Name",
+          "location": "Location",
```

Comment:
```suggestion
          "location": "[%key:common::config_flow::data::location%]",
```

Reply:
Is there somewhere I could find all the common translation keys?